pkg/pipeline/transform/kubernetes/kubernetes.go

```diff
@@ -291,23 +294,13 @@ func (k *KubeData) initServiceInformer(informerFactory informers.SharedInformerF
 	return nil
 }
 
-func (k *KubeData) initReplicaSetInformer(informerFactory informers.SharedInformerFactory) error {
-	k.replicaSets = informerFactory.Apps().V1().ReplicaSets().Informer()
-	// To save space, instead of storing a complete *appvs1.Replicaset instance, the
-	// informer's cache will store a *metav1.ObjectMeta with the minimal required fields
-	if err := k.replicaSets.SetTransform(func(i interface{}) (interface{}, error) {
-		rs, ok := i.(*appsv1.ReplicaSet)
-		if !ok {
-			return nil, fmt.Errorf("was expecting a ReplicaSet. Got: %T", i)
-		}
-		return &metav1.ObjectMeta{
-			Name:            rs.Name,
-			Namespace:       rs.Namespace,
-			OwnerReferences: rs.OwnerReferences,
-		}, nil
-	}); err != nil {
-		return fmt.Errorf("can't set ReplicaSets transform: %w", err)
-	}
+func (k *KubeData) initReplicaSetInformer(informerFactory metadatainformer.SharedInformerFactory) error {
+	k.replicaSets = informerFactory.ForResource(
+		schema.GroupVersionResource{
+			Group:    "extensions",
+			Version:  "v1beta1",
+			Resource: "deployments",
+		}).Informer()
```

Comment:
I don't get this: we need to get a Replicaset informer, not a Deployment?

Reply:
changed to rs